docs/_sass/_main.scss

```diff
@@ -130,6 +134,24 @@ textarea {
     color: $color-dark;
 }
 
+button {
+    border-radius: 12px;
+    padding: 12px;
+    font-family: "GTAmericaExp-Bold", "Helvetica Neue", "Helvetica", Arial, sans-serif;
```

Comment:
Going to skip this NAB just for the sake of my own time-management. Also I think we would have to use a mixin instead of a variable to include all the fallbacks